locals.tf

```diff
@@ -0,0 +1,17 @@
+# ------------------------------------------------------------------------------
+# Retrieve the effective Account ID, User ID, and ARN in which Terraform is
+# authorized.  This is used to calculate the session names for assumed roles.
+# ------------------------------------------------------------------------------
+data "aws_caller_identity" "current" {}
+
+# ------------------------------------------------------------------------------
+# Evaluate expressions for use throughout this configuration.
+# ------------------------------------------------------------------------------
+locals {
+  # Extract the user name of the current caller for use
+  # as assume role session names.
+  caller_user_name = split("/", data.aws_caller_identity.current.arn)[1]
+
+  # Hardcoded value to provide the name for the vmimport service role.
```

Comment:
Can this comment be expanded to make it clear that [AWS requires the role to be named `vmimport`](https://docs.aws.amazon.com/vm-import/latest/userguide/vmie_prereqs.html#vmimport-role) or else the VM import process will not work?

Reply:
Expanded in af8038a. Please let me know if you have any further suggestions.